marklogic-data-hub/src/main/resources/hub-internal-artifacts/flows/default-map-and-master.flow.json

```diff
@@ -1,27 +1,30 @@
 {
-  "name" : "default-map-and-master",
-  "description" : "This is the default flow for mapping previously ingested data and then mastering it into data hub",
+  "name": "default-map-and-master",
+  "description": "This is the default flow for mapping previously ingested data and then mastering it into data hub",
   "options": {
-    "sourceQuery" : "cts.uris(null, null, cts.collectionQuery('default-ingestion'))"
+    "sourceQuery": "cts.uris(null, null, cts.collectionQuery('default-ingestion'))"
   },
-    "steps" : {
-	  "1": {	 
-	   "stepDefinitionType": "mapping",
-	   "stepDefinitionName": "default-mapping",
-	   "retryLimit": 0,
-	   "options": {
-		"outputFormat" : "json",
-		"collections" : "default-mapping"
-		}
-	  },
-	   "2": {	 
-	   "stepDefinitionType": "mastering",
-	   "stepDefinitionName": "default-mastering",
-	   "retryLimit": 0,
-	   "options": {
-		"outputFormat" : "json",
-		"collections" : ["default-mastering", "mastered"]
-		}
-	  }
-	}
+  "steps": {
+    "1": {
+      "stepDefinitionType": "mapping",
+      "stepDefinitionName": "default-mapping",
+      "retryLimit": 0,
+      "options": {
+        "outputFormat": "json",
+        "collections": "default-mapping"
```

Comment:
Can we have the collections as an array to make it consistent across all the flow and step def artefacts

Reply:
Sure can.